pkg/curatedpackages/bundle.go

```
@@ -119,3 +121,18 @@ func (b *BundleReader) UpgradeBundle(ctx context.Context, controller *packagesv1
 	}
 	return nil
 }
+
+func GetPackageBundleRef(vb releasev1.VersionsBundle) string {
+	packageController := vb.PackageController
+	// Use package controller registry to fetch packageBundles.
+	// Format of controller image is: <uri>/<env_type>/<repository_name>
+	controllerImage := strings.Split(packageController.Controller.Image(), "/")
+	major, minor, err := parseKubeVersion(vb.KubeVersion)
+	if err != nil {
+		logger.MarkFail("unable to parse kubeversion", "error", err)
```

curious, why not return an error?